src/main/java/org/radarbase/management/service/config/OAuthClientServiceConfig.kt

```diff
@@ -0,0 +1,58 @@
+package org.radarbase.management.service.config
+
+import org.radarbase.management.config.ManagementPortalProperties
+import org.radarbase.management.service.DefaultOAuthClientService
+import org.radarbase.management.service.OAuthClientService
+import org.radarbase.management.service.HydraOAuthClientService
+import org.radarbase.management.service.mapper.ClientDetailsMapper
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Primary
+import org.springframework.security.oauth2.config.annotation.web.configuration.AuthorizationServerEndpointsConfiguration
+import org.springframework.security.oauth2.provider.client.JdbcClientDetailsService
+
+@Configuration
+class OAuthClientServiceConfiguration {
+
+    @Bean
+    @Primary
+    fun oAuthClientService(
```

Comment:
Better break this up into two conditionally created (using @AuthenticationServer annotation) beans `defaultOAuthClientService` and `hydraOAuthClientService`